app/code/Magento/InventoryCatalog/view/adminhtml/web/template/action-delete.html

```diff
@@ -0,0 +1,18 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<button class="action-delete"
+        attr="{'data-action': 'remove_row'}"
+        disabled="disabled"
+        data-bind="
```

Comment:
Please, avoid using data-bind. Use binding directly through attributes. Example:
```html
<button 
    click="$data.deleteRecord.bind($data, $record().index, $record().recordId)"
    attr="'data-action': 'remove_row', title: $parent.deleteButtonLabel">
</button>
```

Reply:
Hello @omiroshnichenko ,
that code is coming from Magento UI "actionDelete" component for dynamic rows, I just added the disabled binding. Should I change it anyway?

Reply:
@phoenix128 Yes, you introduce new template and it would be nice to have it in correct style.

Reply:
Right, it seems fair.